Example/PrebidDemo/PrebidDemoSwift/NativeController.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  NativeController.swift
+//  PrebidDemoSwift
+//
+//  Created by Punnaghai Puviarasu on 10/14/19.
+//  Copyright © 2019 Prebid. All rights reserved.
+//
+
+import UIKit
+
+import PrebidMobile
+
+import GoogleMobileAds
+
+import MoPub
+
+class NativeController: UIViewController, GADBannerViewDelegate, MPAdViewDelegate {
+    
+    var nativeUnit: NativeRequest!
+    
+    var adServerName: String = ""
+    
+    @IBOutlet var nativeView: UIView!
+    
+    var eventTrackers:NativeEventTracker!
+    var dfpNativeAdUnit:DFPBannerView!
+    var mopubNativeAdUnit:MPAdView!
+    let request = DFPRequest()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        loadNativeAssets()
+        
+        if (adServerName == "DFP") {
+            print("entered \(adServerName) loop" )
+            loadDFPNative()
+
+        } else if (adServerName == "MoPub") {
+            print("entered \(adServerName) loop" )
+            loadMoPubNative()
+
+        }
+        
+        // Do any additional setup after loading the view.
+    }
+    
+    func loadNativeAssets(){
+        let asset1 = NativeAsset()
```

Comment:
why not let asset1 = NativeImageAsset()
let asset2 = NativeTitleAsset()
let asset3 = NativeDataAsset()

Reply:
Changed the NativeAsset implementation to include only 1 object